Meadowlark-js/backends/meadowlark-mongodb-backend/src/model/MeadowlarkDocument.ts

```diff
@@ -47,6 +54,25 @@ export interface MeadowlarkDocument extends MeadowlarkDocumentId {
    */
   outRefs: string[];
 
+  /**
+   * An array of ids this document will satisfy when reference validation performs existence checks.
+   * This array always includes the document id. If this document is a subclass, the array will also
+   * contain the id of this document in superclass form (superclass name and project, identity property
+   * naming differences - like schoolId versus educationOrganizationId - accounted for).
+   *
+   * The idea is that a subclass document provides the existence validation of both the document as its given type
+   * AND as a reference to the superclass when the identity is equivalent.
+   *
+   * Example: Assessment has a reference to EducationOrganization, and ClassPeriod has a reference to School.
+   *          School is a subclass of EducationOrganization, with schoolId being the identity of School and
+   *          educationOrganizationId being the corresponding identity of EducationOrganization.
+   *
+   *          As a result, a single School document with schoolId=123 will satisfy the validation of both
+   *          a reference from ClassPeriod to a School with schoolId=123, as well as a reference from
+   *          Assessment to EducationOrganization with educationOrganizationId=123.
+   */
+  existenceIds: string[];
+
```

Comment:
Description of the array in MongoDB now used for existence checking for reference validation